lib/active_support/cache/redis_store.rb

```diff
@@ -103,12 +103,14 @@ def read_multi(*names)
         args = [keys, options]
         args.flatten!
 
-        values = with { |c| c.mget(*args) }
-        values.map! { |v| v.is_a?(ActiveSupport::Cache::Entry) ? v.value : v }
+        instrument(:read_multi, names) do |payload|
+          values = with { |c| c.mget(*args) }
+          values.map! { |v| v.is_a?(ActiveSupport::Cache::Entry) ? v.value : v }
 
-        result = Hash[names.zip(values)]
-        result.reject!{ |k,v| v.nil? }
-        result
+          Hash[names.zip(values)].reject{|k,v| v.nil?}.tap do |result|
```

Comment:
ah, that's correct. I think it's in Ruby core now but we still support some older versions, so we won't be able to use it right now.